Updates readme (#32)
RascalSoftware/RAT-Docs: README.md

    conda activate RAT
    pip install -r requirements.txt

Download the appropriate version of RAT from the GitHub [release](https://github.com/RascalSoftware/RAT/releases) page, and unzip the contents into a folder called API (This folder should be in the root directory). For example on a Linux machine, the nightly can be downloaded as shown 

    wget https://github.com/RascalSoftware/RAT/releases/download/nightly/Linux.zip
    unzip Linux.zip -d API/

> To build the html docs, in the terminal with access to the python executable, type  

Reviewer: ```suggestion
To build the HTML docs, type the following into a terminal with access to the Python executable:  
```
makes the sentence a bit simpler to read

    make html

> matlabengine is required to generate MATLAB code snippet outputs. If matlabengine is not installed the outputs will be omitted and the following warning will be printed in the terminal

Reviewer: ```suggestion
[`matlabengine`](https://pypi.org/project/matlabengine/) is required to generate MATLAB code snippet outputs. If `matlabengine` is not installed, the outputs will be omitted and the following warning will be printed in the terminal:
```

    UserWarning: Could not create output as MATLAB engine was not available

> If the MATLAB code outputs are needed, install the appropriate matlabengine for your installed MATLAB

Reviewer: ```suggestion
If the MATLAB code outputs are needed, install the appropriate `matlabengine` for your installed MATLAB
```
